sed/config/flash_example_config.yaml

```diff
@@ -21,8 +23,16 @@ core:
     data_parquet_dir: "tests/data/loader/flash/parquet"
 
 binning:
-  # Since this will run on maxwell most probably, we have a lot of cores at our disposal
-  num_cores: 100
+  # Histogram computation mode to use.
+  hist_mode: "numba"
```

Comment:
why not have these in the default config instead of each loader specific config?

Reply:
Effectively they are the defaults. They are just there to showcase which config parameters are available.